.gitignore

```diff
@@ -1,6 +1,4 @@
 npm-debug.log
 node_modules
 build
-dist
-lib
-
+.idea
```

Comment:
This should be added by IntelliJ users in their own global `~/.gitignore` file. Also please readd ignore rules for `dist` and `lib` (see contextual comments below).